contracts/GroupHealth.sol

```diff
@@ -0,0 +1,297 @@
+     * @param group The group's address.
+     * @param realCelo The real Celo value.
+     * @param expectedCelo The expected Celo value.
+     */
+    error RebalanceEnoughCelo(address group, uint256 realCelo, uint256 expectedCelo);
+
+    /**
+     * @notice Initialize the contract with registry and owner.
+     * @param _registry The address of the Celo registry.
+     * @param _owner The address of the contract owner.
+     */
+    function initialize(address _registry, address _owner) external initializer {
+        _transferOwnership(_owner);
+        __UsingRegistry_init(_registry);
+    }
+
+    /**
+     * @notice Set this contract's dependencies in the StakedCelo system.
+     * @dev Manager, Account and StakedCelo all reference each other
+     * so we need a way of setting these after all contracts are
+     * deployed and initialized.
+     * @param _stakedCelo the address of the StakedCelo contract.
+     * @param _account The address of the Account contract.
+     * @param _specificGroupStrategy The address of the SpecificGroupStrategy contract.
+     * @param _defaultStrategy The address of the DefaultStrategy contract.
+     * @param _manager The address of the Manager contract.
+     */
+    function setDependencies(
+        address _stakedCelo,
+        address _account,
+        address _specificGroupStrategy,
+        address _defaultStrategy,
+        address _manager
+    ) external onlyOwner {
+        require(_stakedCelo != address(0), "StakedCelo null");
+        require(_account != address(0), "Account null");
+        require(_specificGroupStrategy != address(0), "SpecificGroupStrategy null");
+        require(_defaultStrategy != address(0), "DefaultStrategy null");
+        require(_manager != address(0), "Manager null");
+
+        stakedCelo = IStakedCelo(_stakedCelo);
+        account = IAccount(_account);
+        specificGroupStrategy = ISpecificGroupStrategy(_specificGroupStrategy);
+        defaultStrategy = IDefaultStrategy(_defaultStrategy);
+        manager = IManager(_manager);
+    }
+
+    /**
+     * @notice Returns the storage, major, minor, and patch version of the contract.
+     * @return Storage version of the contract.
+     * @return Major version of the contract.
+     * @return Minor version of the contract.
+     * @return Patch version of the contract.
+     */
+    function getVersionNumber()
+        external
+        pure
+        returns (
+            uint256,
+            uint256,
+            uint256,
+            uint256
+        )
+    {
+        return (1, 1, 0, 0);
+    }
+
+    /**
+     * @notice Checks if a group meets the validator group health requirements.
+     * @param group The group to check for.
+     * @return Whether or not the group is valid.
+     */
+    function isValidGroup(address group) public view returns (bool) {
+        IValidators validators = getValidators();
+
+        // add check if group is !registered
```

Comment:
Is this comment saying the check still needs to be added? If so, should the check be added in this PR? If not, is the comment useful?

Reply:
Personally would prefer full wording, "not registered".